apps/parsley/src/hooks/useSections/useSections.test.tsx

```diff
@@ -305,13 +305,13 @@ describe("useSections", () => {
     });
   });
 
-  it("should open the section containing 'onInitOpenSectionContainingLine' during initialization only", async () => {
+  it("should open the section containing 'onInitOpenSectionsContainingLines' during initialization only", async () => {
```

Comment:
Should a test be added with an `onInitOpenSectionsContainingLines` array of > 1 element?

Reply:
I updated at test to include an array with length > 1. I test this further in populateSectionState tests [here](https://github.com/evergreen-ci/ui/pull/333/files#diff-a2ce52fa21af9fc02b6ed9726c865b4e6a7694d4c2fb65309a77af3feb73407bR572)